src/Microsoft.AspNetCore.BuildTools.ApiCheck/Program.cs

```diff
@@ -186,65 +205,160 @@ public static int Main(string[] args)
+                .ToList();
+            if (newMembersOnInterfaces.Count > 0)
+            {
+                if (!compactOutput)
+                {
+                    Console.WriteLine();
+                    Console.WriteLine();
+                    Console.WriteLine("Error: Following interfaces have one or more members added from them.");
+                    Console.WriteLine();
+                }
+
+                foreach (var memberGrouping in newMembersOnInterfaces)
+                {
+                    if (compactOutput)
+                    {
+                        Console.WriteLine(string.Join(Environment.NewLine,
+                            memberGrouping.Select(
+                                m => $"Error: Missing member on type '{m.TypeId}' => '{m.MemberId}'")));
+                    }
+                    else
+                    {
+                        Console.WriteLine(indent + memberGrouping.Key);
+                        Console.WriteLine();
+                        Console.WriteLine(string.Join(Environment.NewLine,
+                            memberGrouping.Select(b => indent + indent + b.MemberId).ToList()));
+                        Console.WriteLine();
+                    }
+                }
+            }
+
+            breakingChangesToPrint.AddRange(newMembersOnInterfaces.SelectMany(t => t.ToList()));
+
+            foreach (var exclusion in incorrectBreakingChanges)
             {
                 if (compactOutput)
                 {
-                    Console.WriteLine($"ERROR: Exclusion is no longer necessary: Old type '{exclusion.OldTypeId}' member " +
-                        $"'{exclusion.OldMemberId}, New type '{exclusion.NewTypeId}' member '{exclusion.NewMemberId}'");
+                    Console.WriteLine($"ERROR: Exclusion is no longer necessary: Type '{exclusion.TypeId}' Member " +
+                                      $"'{exclusion.MemberId}'");
                 }
                 else
                 {
-                    Console.WriteLine($@"ERROR: The following exclusion is in the exclusion file, but is no longer necessary:
-    Old type: {exclusion.OldTypeId}
-    Old member: {exclusion.OldMemberId}
-    New type: {exclusion.NewTypeId}
-    New member: {exclusion.NewMemberId}");
-
+                    Console.WriteLine(
+                        "ERROR: The following exclusion is in the exclusion file, but is no longer necessary:");
+                    Console.WriteLine(JsonConvert.SerializeObject(incorrectBreakingChanges, Formatting.Indented));
                     Console.WriteLine();
                 }
             }
 
-            if (differences.Count > 0 || remainingExclusions.Count > 0)
+            if (!compactOutput && breakingChangesToPrint.Any())
+            {
+                Console.WriteLine();
+                Console.WriteLine();
+                Console.WriteLine("Following is the list of exclusions which are needed to be added.");
```

Comment:
`Following is the list of exclusions that either need to be added to the list of breaking changes, or the breaking changes themselves need to be reverted:`